### ISSUE TITLE
public webminerpool server

### ISSUE THREAD

**Issue by notgiven688** [notgiven688/webminerpool#57]
Collective thread for people running webminerpool servers which are available for everyone. Please post your server and specs here.

**Comment by nadal28** [comment]
# Network
- **Host**: bubble.solfa.org

- **Connection**: _wss://bubble.solfa.org/wss_

- **Status**: **Will be online today!**
- **Fee**: 1%
# Specs
- **Ram**: 1GB
- **CPU**: 1 core
- **Limits**: No limits, unmetered traffic and unlimited connections

**Comment by PiTi2k5** [comment]
[crypto-webminer.com](url)

Based on webminerpool with some custom implemantions for example: Using your own favorite custom pool, integration for websites/blogs or use my webmining UI. Check it out :)

Mining possible with Browser, Windows 10 UWP app, iOS app or my Android app

**Basic informations:**
6x Highend Root Server for max performance & stability
4x websockets
DDOS protection
Server Locations: Germany with world wide low latency
Fee 1%
Trusted developer: Ref project a-calc.de with over 1.5 million users
Discord [https://discord.gg/nnARdpc](url) & Email info@crypto-webminer.com Support
Webmining service runs since nov17
Constant over 50000+ active miners - Thanks for your trust and support!

**Technical specifications (for each Root Server)**
10 Core Intel® Xeon® Gold 6140 CPU
32 GB DDR4 RAM
120 GB SSD
1 GBit/s Network - unlimted traffic

**Happy mining**

**Comment by GustavoEdny** [comment]
# **www.webminerpools.com**
</hr>

**This version supports as many customers as possible. Your servers can be expanded to any level and support your users traffic. Because it is a newly created project, the rate of 5% is maintained until we get a user number that can help us with such hosting costs. Our goal is to make this project have no fee soon. To compensate for this, you should choose a low fee pool so that the impact on your earnings is lower.**

In addition to **MONERO** other currencies will be supported.

To mine at **AEON** use **https://www.webminerpools.com/aeon/** and follow the instructions.

If you have a suggestion or want another currency or pool to be incorporated, contact us: **support@webminerpools.com**

## Network

- **Host** : webminerpools.com
- **Region** : London
- **Connection** : wss://www.webminerpools.com:8181
- **Status** : online
- **Limits** : unlimited
- **Fee** : 5%

**Comment by brainscar** [comment]
@PiTi2k5 your first link is broken, just thought I'd let you know.

**Comment by paul0921** [comment]
Check out https://minexmr.stream. minexmr.stream has a 5% fee and is automatically scaled with demand. For large websites, a lower cut may be negotiated.

**Comment by paul0921** [comment]
@notgiven688 I own https://minexmr.stream, and I request that @GustavoEdny 's server be removed from this thread, as the content of the site was copied and altered to fit his need without express permission.

**Comment by notgiven688** [comment]
@paul0921 You both copied my original work and ignore the MIT license which clearly states

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

*This message will be deleted soon*

**Comment by PiTi2k5** [comment]
How old are you @paul0921 ? Copyright on 2 sentences??? Really? xD

**Comment by paul0921** [comment]
@PitTi2k5 I agree it's rather a minority, however this person, if it's only "two sentences", as you say, should have written them independently. If someone has claimed their exclusive right on some content, it should not be infringed upon.

**Comment by GustavoEdny** [comment]
Hello @paul0921, it is not necessary to remove my comment. I apologize for any inconvenience. the front end you are referring to are being remade and I will no longer use such a code. just used to give an initial tip to foot on my server, did not know that it belonged to you. Once again I apologize for the inconvenience. My goal is just to keep this project running.

**Comment by paul0921** [comment]
Thanks 👍

**Comment by VidYen** [comment]
**VYPS WordPress Plugin Miner**

Not actually a pure webminerpool as we forked the code for our own use with extra options, but its free and open source client in WordPress so you can actually award points to reward users for mining so they can buy virtual items or whatever you want to sell in WooCommerce.

[VYPS](https://wordpress.org/plugins/vidyen-point-system-vyps/)

With the servers themselves are at:

wss://cloud.vy256.com:8181

wss://vy256.com:8181

Yes. 	notgiven688, you are technically on the donation list on the server we run at **3%** as I figure you will do most of the work come next fork of Monero, but we added our own wallet for **6%** on top of that (which you might want to just add that option in your code out of the box so people start a chain of donations down the forks).

So fees are **9%** between me and notgiven688.

So to explain, we added some code that lets us track users by hash servers side so that it can be used for a rewards site rather than blanket silent mining. As users mine, the WordPress site does a curl to see how many hashes the current user had and then award them points based on that.

We actually run more than one server so the plugin will check to see which one is up and then switch to that ( wss://vy256.com and wss://cloud.vy256.com ) and although you can connect directly (which as I don't mind as our donations are all server side) but I would recommend to use the WP plugin so you get the full experience.

And if you do download the plugin we have Adscend and Coinhive options, but the WMP is the VY256 shortcode which is also better at avoiding ad blockers.

Currently cloud.vy256.com is:

**g1-small
 Shared-core machine type with 0.5 vCPU
1.70 GB of system memory, backed by a shared physical core.** 

Running Debian.

Not that much, but since it's GCP, I can just adjust as needed.

Also as an aside.... The VYPS plugin has built in raffle game, leader board, and conversion system to WooCommerce credit as well as a miner sharing system. The miner sharing system allows users to put their own XMR address to get a change at having other users mine to their wallets for them. That part requires WordPress as need to use their back end to store user meta, but its a way to have users compete for mining or do things like watch ads for points to indirectly earn XMR. In cases where no users wins the pool roll (happens during each time client side miner is loaded) the house wallet wins.

If you need more visuals (rather than my wall of text), I also have this [Mine to Pay](https://youtu.be/O7CCGP5vYRA) tutorial video of how this works on a sample WordPress site.

**Comment by jhabdas** [comment]
@VidYen could you do us a favor and update your "Mine to Pay" link so it's obvious it goes to YouTube. I just inadvertently let Google Track me after clicking on the link not realizing it led to an adware site.

**Comment by jhabdas** [comment]
https://after-dark.habd.as/module/toxic-swamp/

**Comment by jhabdas** [comment]
Herein I describe a swarm of public proxy servers I'll be creating soon to accompany the push-button miner I created (see linked video demo for example): https://after-dark.habd.as/module/toxic-swamp/

The swarm will be portable to any Docker Swarm node including a Raspberry Pi and will reward users using a heuristics-based approach to determine if they've been keeping their software up-to-date.

If you like this please consider upvoting in the following locations:

https://www.reddit.com/r/Monero/comments/9mfrva/complete_sources_for_a_pushbutton_cryptonote/

https://news.ycombinator.com/item?id=18152650

**Comment by jhabdas** [comment]
I have a pseudo-public server up which intends to gamify the software upgrade process using a public server created from the awesome docker image. To see it running and download the source code, which adds a sophisticated Web UI please head here for sources: https://after-dark.habd.as/module/toxic-swamp/

**Comment by Lillythefox** [comment]
moneroocean.stream has no fees

**Comment by barrettathome** [comment]
Running `wss://f.xmrminingproxy.com:8181/`

12% fee but that doesn't cover the server.  SOLO mining MSR far as donations go.  Luckily I have other revenue sources to cover the server cost.

We also offer MSR, SUMO, and LTHN mining.
`masari.hashvault.pro`
`sumokoin.hashvault.pro`
`lethean.hashvault.pro`

**Comment by 2Peti** [comment]
i am running `wss://webminer.cochces.eu/`

Only 1% fee.
